### ISSUE TITLE
Notifications Page UI

### ISSUE THREAD

**Issue by shreya024** [shreya024/MemoriesApp#49]
Develop a UI for the notifications page having similar style as the Login page.

**Comment by codingmickey** [comment]
What is expected to be in this page?

**Comment by shreya024** [comment]
> What is expected to be in this page?

Notifications for likes, shares, followers or new post alerts from accounts the user is following...similar to Twitter or LinkedIn Notifications

**Comment by fykaa** [comment]
@codingmickey Are you an SSOC participant interested in getting assigned with this issue?

**Comment by Soumava-221B** [comment]
Hello i am a participant of SSOC, can you assign me this issue?

**Comment by shreya024** [comment]
> Hello i am a participant of SSOC, can you assign me this issue?

Any updates?

**Comment by Soumava-221B** [comment]
Still working on the notification page. I have doubt the notification page would look like this 
![Screenshot from 2022-08-22 10-21-48](https://user-images.githubusercontent.com/77767745/185842090-1469ad33-feaf-4d78-81a0-1299ae01f31c.png)

**Comment by shreya024** [comment]
@Soumava-221B yup looks good so far.Also have to do how each notification will look

**Comment by Soumava-221B** [comment]
![Screenshot from 2022-08-29 23-53-33](https://user-images.githubusercontent.com/77767745/187271900-4c65892a-4f33-41c7-90ed-7448437f52c2.png)
This how each notification would look like

**Comment by Soumava-221B** [comment]
@shreya024 After running in the memories app notification page looks like this
![Screenshot from 2022-09-01 20-16-37](https://user-images.githubusercontent.com/77767745/187944296-28a095e3-f9c3-481c-8182-c17a7c29d446.png)

**Comment by shreya024** [comment]
> @shreya024 After running in the memories app notification page looks like this ![Screenshot from 2022-09-01 20-16-37](https://user-images.githubusercontent.com/77767745/187944296-28a095e3-f9c3-481c-8182-c17a7c29d446.png)

ok.But fix the alignment and make sure it is responsive and then make a PR to the ssoc branch

**Comment by Debdeep1** [comment]
@Soumava-221B make the PR if you have finished working on the fixes.

**Comment by Soumava-221B** [comment]
@shreya024 Responsive? Can you elaborate
![Screenshot from 2022-09-08 10-11-59](https://user-images.githubusercontent.com/77767745/189035923-e06219de-6ca9-46fd-9549-3014ef60e9ea.png)
